ui/app/components/clients/history.js

```diff
@@ -193,38 +185,58 @@ export default class History extends Component {
     return this.queriedActivityResponse || this.args.model.activity;
   }
 
+  get byMonthTotalClients() {
+    return this.getActivityResponse?.byMonth;
```

Comment:
This situation is a little confusing because you use this.getActivityResponse in the definition of a tracked property but getActivityResponse itself is not a tracked property but is used to define a getter. Normally any property in a getter I add as a tracked property, but I'm not sure in this case. Just throwing it out there as maybe something to test. Re-query the date range and put a console log in this getter and make sure it fires when getActivityResponse changes.

Reply:
Yep! It all updates. `this.getActivityResponse` changes anytime there is a network request. My understanding is that tracked properties are best used for state changes from the user? 

Also confirmed with console logs all works as expected!